Logs too noisy when there is a missing image in shortyservlet

### Problem Statement

There is too much logging when a page requests an image by shorty that does not exist.

This was done in the changes here: #29851

```
14:37:09.957  ERROR servlet.VisitorFilter - java.lang.NullPointerException: Cannot invoke "com.dotmarketing.portlets.contentlet.model.Contentlet.getInode()" because "imageContentlet" is null
javax.servlet.ServletException: java.lang.NullPointerException: Cannot invoke "com.dotmarketing.portlets.contentlet.model.Contentlet.getInode()" because "imageContentlet" is null
	at com.dotmarketing.servlets.ShortyServlet.service(ShortyServlet.java:94) ~[?:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:623) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:209) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotmarketing.filters.CMSFilter.doFilterInternal(CMSFilter.java:198) ~[?:?]
	at com.dotmarketing.filters.CMSFilter.doFilter(CMSFilter.java:59) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotcms.visitor.filter.servlet.VisitorFilter.doFilter(VisitorFilter.java:81) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotcms.vanityurl.filters.VanityURLFilter.doFilter(VanityURLFilter.java:107) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at org.tuckey.web.filters.urlrewrite.RuleChain.handleRewrite(RuleChain.java:176) ~[?:?]
	at org.tuckey.web.filters.urlrewrite.RuleChain.doRules(RuleChain.java:145) ~[?:?]
	at org.tuckey.web.filters.urlrewrite.UrlRewriter.processRequest(UrlRewriter.java:92) ~[?:?]
	at org.tuckey.web.filters.urlrewrite.UrlRewriteFilter.doFilter(UrlRewriteFilter.java:389) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotmarketing.filters.TimeMachineFilter.doFilter(TimeMachineFilter.java:137) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotmarketing.filters.ThreadNameFilter.doFilter(ThreadNameFilter.java:88) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotmarketing.filters.CookiesFilter.doFilter(CookiesFilter.java:53) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotmarketing.filters.CharsetEncodingFilter.doFilter(CharsetEncodingFilter.java:99) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at org.apache.catalina.filters.HttpHeaderSecurityFilter.doFilter(HttpHeaderSecurityFilter.java:129) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at com.dotcms.filters.NormalizationFilter.doFilter(NormalizationFilter.java:89) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:178) ~[?:?]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:153) ~[?:?]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:168) ~[?:?]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90) ~[?:?]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:481) ~[?:?]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:130) ~[?:?]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:670) ~[?:?]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93) ~[?:?]
	at org.apache.catalina.valves.RemoteIpValve.invoke(RemoteIpValve.java:765) ~[?:?]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74) ~[?:?]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342) ~[?:?]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:390) ~[?:?]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63) ~[?:?]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:928) ~[?:?]
	at org.apache.tomcat.util.net.Nio2Endpoint$SocketProcessor.doRun(Nio2Endpoint.java:1732) ~[?:?]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52) ~[?:?]
	at org.apache.tomcat.util.net.AbstractEndpoint.processSocket(AbstractEndpoint.java:1295) ~[?:?]
	at org.apache.tomcat.util.net.Nio2Endpoint.setSocketOptions(Nio2Endpoint.java:339) ~[?:?]
	at org.apache.tomcat.util.net.Nio2Endpoint$Nio2Acceptor.completed(Nio2Endpoint.java:479) ~[?:?]
	at org.apache.tomcat.util.net.Nio2Endpoint$Nio2Acceptor.completed(Nio2Endpoint.java:415) ~[?:?]
	at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:129) ~[?:?]
	at java.base/sun.nio.ch.Invoker.invokeDirect(Invoker.java:160) ~[?:?]
	at java.base/sun.nio.ch.Invoker.invoke(Invoker.java:188) ~[?:?]
	at java.base/sun.nio.ch.UnixAsynchronousServerSocketChannelImpl.onEvent(UnixAsynchronousServerSocketChannelImpl.java:194) ~[?:?]
	at java.base/sun.nio.ch.KQueuePort$EventHandlerTask.run(KQueuePort.java:312) ~[?:?]
	at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "com.dotmarketing.portlets.contentlet.model.Contentlet.getInode()" because "imageContentlet" is null
	at com.dotmarketing.servlets.ShortyServlet.inodePath(ShortyServlet.java:543) ~[?:?]
	at com.dotmarketing.servlets.ShortyServlet.doForward(ShortyServlet.java:348) ~[?:?]
	at com.dotmarketing.servlets.ShortyServlet.serve(ShortyServlet.java:294) ~[?:?]
	at com.dotmarketing.servlets.ShortyServlet.service(ShortyServlet.java:92) ~[?:?]
	... 76 more
```

### Steps to Reproduce

To reproduce, hit the home page of the demo starter.



### Acceptance Criteria

There should be no logging here by default.  It should be debug I think

### dotCMS Version

24.10

### Proposed Objective

Application Performance

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

--- a/dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java
+++ b/dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java
@@ -539,10 +539,17 @@ public class ShortyServlet extends HttpServlet {
 
             if (contentletVersionInfo.isPresent()) {
                 Logger.debug(this, "Contentlet version found for identifier: " + relatedImageId);
-                final Contentlet imageContentlet = getImageContentlet(contentletVersionInfo.get(), live);
-                validateContentlet(imageContentlet, live, imageContentlet.getInode());
-                final String fieldVar = imageContentlet.isDotAsset() ? DotAssetContentType.ASSET_FIELD_VAR : FILE_ASSET_DEFAULT;
-                return buildFieldPath(imageContentlet, fieldVar);
+                final String inode = live
+                        ? contentletVersionInfo.get().getLiveInode()
+                        : contentletVersionInfo.get().getWorkingInode();
+                try{
+                    final Contentlet imageContentlet = APILocator.getContentletAPI().find(inode, APILocator.systemUser(), false);
+                    validateContentlet(imageContentlet, live, inode);
+                    final String fieldVar = imageContentlet.isDotAsset() ? DotAssetContentType.ASSET_FIELD_VAR : FILE_ASSET_DEFAULT;
+                    return buildFieldPath(imageContentlet, fieldVar);
+                }catch (DotDataException e){
+                    Logger.debug(this.getClass(), e.getMessage());
+                }
             }
             Logger.debug(this, "No contentlet version found for identifier: " + relatedImageId + ", returning path based on original contentlet inode: " + contentlet.getInode());
         }
@@ -560,22 +567,6 @@ public class ShortyServlet extends HttpServlet {
     private boolean shouldFallbackToDefaultLanguage(final Contentlet contentlet) {
         return APILocator.getLanguageAPI().canDefaultFileToDefaultLanguage() &&
                 APILocator.getLanguageAPI().getDefaultLanguage().getId() != contentlet.getLanguageId();
-    }
-
-    /**
-     * Retrieves the appropriate contentlet version (live or working) for an image
-     * based on the provided version info.
-     *
-     * @param versionInfo The version information for the contentlet
-     * @param live Whether to retrieve the live version (true) or working version (false)
-     * @return The requested version of the contentlet
-     * @throws DotDataException If there's an error accessing the data
-     * @throws DotSecurityException If there's a security violation
-     */
-    private Contentlet getImageContentlet(final ContentletVersionInfo versionInfo, final boolean live) 
-            throws DotDataException, DotSecurityException {
-        final String inode = live ? versionInfo.getLiveInode() : versionInfo.getWorkingInode();
-        return APILocator.getContentletAPI().find(inode, APILocator.systemUser(), false);
     }
 
     /**

